dask_ml/decomposition/incremental_pca.py

```diff
@@ -135,12 +136,18 @@ def __init__(
     ):
         self.n_components = n_components
         self.whiten = whiten
+        self.center = center
         self.copy = copy
         self.batch_size = batch_size
         self.svd_solver = svd_solver
         self.iterated_power = iterated_power
         self.random_state = random_state
 
+    def _check_params(self):
```

Comment:
I've opted for `_check_params` (as opposed to `_check_parameters`), because that's what the majority choice has been:
```bash
$ grep -rnHwc --color=never "def _check_params" . --include=\*.py | grep -v ":0"                                                                                                                                                                                                                                
./sklearn/feature_selection/_univariate_selection.py:4
./sklearn/feature_extraction/text.py:1
./sklearn/cluster/_kmeans.py:2
./sklearn/neighbors/_graph.py:1
./sklearn/decomposition/_lda.py:1
./sklearn/decomposition/_nmf.py:1
./sklearn/ensemble/_gb.py:1

$ grep -rnHwc --color=never "def _check_parameters" . --include=\*.py | grep -v ":0"                                                                                                                                                                                                                            
./sklearn/mixture/_bayesian_mixture.py:1
./sklearn/mixture/_base.py:1
./sklearn/mixture/_gaussian_mixture.py:1
./sklearn/cluster/_bicluster.py:2
```